internal/kernel/t_api/status.go

```diff
@@ -17,9 +17,12 @@ const (
 	StatusPromiseAlreadyRejected ResponseStatus = 4031
 	StatusPromiseAlreadyCanceled ResponseStatus = 4032
 	StatusPromiseAlreadyTimedOut ResponseStatus = 4033
+	StatusLockAlreadyAcquired    ResponseStatus = 4034
 	StatusPromiseNotFound        ResponseStatus = 4040
 	StatusSubscriptionNotFound   ResponseStatus = 4041
 	StatusScheduleNotFound       ResponseStatus = 4042
+	StatusLockNotFound           ResponseStatus = 4043
+	StatusProcessIdHasNoLocks    ResponseStatus = 4044
```

Comment:
I'm torn between leaving this as a separate response or just reusing `StatusLockNotFound`. What were the reasons you wanted to use a separate response code?